libraries/botbuilder-core/package.json

```diff
@@ -21,7 +21,8 @@
   "typings": "./lib/index.d.ts",
   "dependencies": {
     "assert": "^1.4.1",
-    "botframework-schema": "4.1.6"
+    "botframework-schema": "4.1.6",
```

Comment:
What is the exact change that forces this dependency?

Reply:
TokenExchangeResource is auto generated form swagger and lives in botframework-connector. Since TokenExchangeResource is part of the oauth card and is used in creation of oauthcard in cardfactory, we need this dependency to import TokenExchangeResource

---
In reply to: [383512980](https://github.com/microsoft/botbuilder-js/pull/1738#discussion_r383512980) [](ancestors = 383512980)